src/site/content/en/mini-apps/mini-app-example-project/index.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: post
+title: An example project
```

Comment:
This title won't get good SEO. Something that mentions that you're applying mini app principles to web development will result in more organic search

Reply:
I like your proposal! Changed the title to it.